etl/storage/storage_test.go

```diff
@@ -8,26 +8,52 @@
 package storage
 
 import (
+	"context"
+	"os"
 	"testing"
 
+	"github.com/jinzhu/gorm"
 	"github.com/stretchr/testify/require"
 
 	"github.com/insolar/block-explorer/etl/models"
+	"github.com/insolar/block-explorer/instrumentation/belogger"
 	"github.com/insolar/block-explorer/testutils"
 )
 
-func TestStorage_SaveJetDropData(t *testing.T) {
+var testDB *gorm.DB
+
+func TestMain(t *testing.M) {
+	var dbCleaner func()
 	var err error
-	testDB, dbCleaner, err := testutils.SetupDB()
-	require.NoError(t, err)
+	testDB, dbCleaner, err = testutils.SetupDB()
+	if err != nil {
+		belogger.FromContext(context.Background()).Fatal(err)
+	}
+	retCode := t.Run()
 	defer dbCleaner()
+	dbCleaner()
```

Comment:
why do you call `dbCleaner()` second time?

Reply:
no reason, delete it